lib/VerifiableCredentialStore.js

```diff
@@ -141,13 +153,13 @@ export default class VerifiableCredentialStore {
       const issuer = trustedIssuers.filter(({required}) => required)
         .map(({issuer}) => issuer);
       const results = await this.find({issuer, type});
-      if(results.length === 0) {
-        const err = new Error(`Unable to find required credential.`);
-        err.name = 'NotFoundError';
-        err.query = example;
-        throw err;
-      }
-      return results
+      // if(results.length === 0) {
+      //   const err = new Error(`Unable to find required credential.`);
+      //   err.name = 'NotFoundError';
+      //   err.query = example;
+      //   throw err;
+      // }
```

Comment:
Let's just remove this commented conditional and have the API return an array of results.

Reply:
OK, I'm assigning this PR to myself to wrap up sometime.